docs/running/notebooks/source_files/program_1.py

```diff
@@ -3,6 +3,7 @@
 
 from quantum_serverless import save_result
 
+print("Running program 1...")
```

Comment:
Should you remove the prints here? :)

Reply:
I actually added these so we would have something short to show in the logs when we show the logs in the tutorials.

I could use the logging package, or I could leave these print statements. What do you think?

Reply:
@akihikokuroda @IceKhan13 do we have some logging package used in other samples? Should we maintain the simple print?

Reply:
as long as it spits to stdio :) 
print is fine, anyway most of the users will do that